packages/ember-htmlbars-template-compiler/lib/index.js

```diff
@@ -1,4 +1,5 @@
 export { default as compile } from './system/compile';
 export { default as precompile } from './system/precompile';
 export { default as template } from './system/template';
+export { default as defaultCompileOptions } from './system/compile-options';
```

Comment:
So that `ember-template-compiler/system/compile-options` can use the existing `../compiler` module to grab all that it needs.  Otherwise, we would have to encode the feature flag in another location...